pontoon/base/static/js/fluent_interface.js

```diff
@@ -252,7 +262,8 @@ var Pontoon = (function (my) {
               item.key.value || item.key.name,
               item.value.elements,
               isPluralElement(element),
-              isTranslated
+              isTranslated,
+              isCustomAttribute
```

Comment:
Nit: you can add trailing commas in function arguments, this way next time you make a change the diff will only show one line instead of three.